### PR TITLE
FAQ: add a TiKV question about the store size

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs-cn/pull/1104/ PTAL @lilin90 @overvenus

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -435,6 +435,10 @@ Not really. We need some good features of gRPC, such as flow control, encryption
 
 The 92 indicates the escape character, which is ASCII 92 by default.
 
+#### Why does the data length shown by `information_schema.tables.data_length` differ from the store size on the TiKV monitoring panel?
+	
+Because the two results are calculated in different ways. `information_schema.tables.data_length` is an estimated value by calculating the averaged length of each row, while the store size on the TiKV monitoring panel sums up the length of the data files (the SST files of RocksDB) in a single TiKV instance. Because of the existence of multiple versions and the data compressed by TiKV, different values are shown in the two results.
```
Comment (lilin90) [inline FAQ.md]:
```suggestion
Two reasons:

- The two results are calculated in different ways. `information_schema.tables.data_length` is an estimated value by calculating the averaged length of each row, while the store size on the TiKV monitoring panel sums up the length of the data files (the SST files of RocksDB) in a single TiKV instance.
- `information_schema.tables.data_length` is a logical value, while the store size is a physical value. The redundant data generated by multiple versions of the transaction is not included in the logical value, while the redundant data is compressed by TiKV in the physical value.
```

Comment (lilin90) [reply FAQ.md]: @overvenus Please help check whether this is what you mean. Thanks!

Comment (overvenus) [reply FAQ.md]: LGTM

Comment (yikeke) [reply FAQ.md]: Updated. PTAL again. @lilin90

**Pull request merged**